src/pages/dashTemplate/sideBar/BentoFilterUtils.js

```diff
@@ -52,3 +60,27 @@ export async function getAllSubjectIds(subjectIdsArray) {
     .catch(() => []);
   return allids;
 }
+
+export const setActiveFilterByPathQuery = (match) => {
+  const query = decodeURI(match.params.filterQuery || '');
```

Comment:
This should be const `const query = decodeURIComponent(match.params.filterQuery || '');` to match the package encoding.